src/route-config.ts

```diff
@@ -97,6 +98,12 @@ export const routes: RoutesFn = (t: TFunction) => [
     breadcrumb: t("roles:roleList"),
     access: "view-realm",
   },
+  {
+    path: "/roles/:id",
+    component: RolesForm,
+    breadcrumb: t("roles:roleDetails"),
+    access: "view-realm",
```

Comment:
The code that processes `route-config.ts` needs to be able to handle multiple roles.  And we don't do anything with subroutes right now anyway so I'm not sure how we need to check them.  We don't have any of that yet, but I believe that you should still be able to code it like this:  
```suggestion
    access: ["view-realm", "view-users"],
```

Reply:
Thank you for your helpful review comments @ssilvert ! this doesn't seem to work at the moment, I get `Forbidden` displayed on the page instead of the details tab. what do you suggest would be an acceptable fix in the meantime? Should I just leave `view-realm`?

Reply:
> Thank you for your helpful review comments @ssilvert ! this doesn't seem to work at the moment, I get `Forbidden` displayed on the page instead of the details tab. what do you suggest would be an acceptable fix in the meantime? Should I just leave `view-realm`?

That's fine.  I'll just make an issue for it.